script/lint.js

```diff
@@ -41,6 +41,32 @@ function spawnAndCheckExitCode (cmd, args, opts) {
   if (status) process.exit(status)
 }
 
+function gnCheckConfig (config, env) {
```

Comment:
i'm inclined to drop this from the script now that it's just a simple `gn check` in CI.

Reply:
Yup I would be in favor of that, wasn't sure if users wanted to a simple one liner to replicate the failure they saw in ci.

Reply:
i think this one-liner will probably cause more problems than it solves. `gn check out/Debug //electron:electron_lib` is already a one-liner :)